src/Sarif.Multitool/Rules/SARIF1004.ExpressUriBaseIdsCorrectly.cs

```diff
@@ -33,11 +37,13 @@ public class ExpressUriBaseIdsCorrectly : SarifValidationSkimmerBase
 
         protected override void Analyze(ArtifactLocation fileLocation, string fileLocationPointer)
         {
-            // UriBaseIdRequiresRelativeUri: The 'uri' property of 'fileLocation' must be a relative uri, since 'uriBaseId' is present.
             if (fileLocation.UriBaseId != null && fileLocation.Uri.IsAbsoluteUri)
             {
-                // {0}: This fileLocation object contains a "uriBaseId" property, which means that the value
-                // of the "uri" property must be a relative URI reference, but "{1}" is an absolute URI reference.
+                // TODO: hakohlli - fix parameters.
```

Comment:
> TODO: hakohlli - fix parameters. [](start = 18, length = 33)

As the code stands, there are not enough arguments in the call to `LogResult` to fill up all the replacement sequences. So why doesn't this crash in the functional test?

Reply:
as discussed offline, we are going to change functionality here so that this test is performed only for originaluribaseids.

---
In reply to: [448531084](https://github.com/microsoft/sarif-sdk/pull/1956#discussion_r448531084) [](ancestors = 448531084)